benchmarks/benchmark.py

```diff
@@ -29,208 +31,230 @@
-				'memory usage 9 (% of memory)', 
-				'memory usage 10 (% of memory)']]
+                'Source Video', 
+                'Image Path', 
+                'Kernel Width (# pixels)', 
+                'Average Time Elapsed (milliseconds)', 
+                'Variance of Time Elapsed (milliseconds)', 
+                'Time Elapsed 1 (milliseconds)', 
+                'Time Elapsed 2 (milliseconds)', 
+                'Time Elapsed 3 (milliseconds)', 
+                'Time Elapsed 4 (milliseconds)', 
+                'Time Elapsed 5 (milliseconds)', 
+                'Time Elapsed 6 (milliseconds)', 
+                'Time Elapsed 7 (milliseconds)', 
+                'Time Elapsed 8 (milliseconds)', 
+                'Time Elapsed 9 (milliseconds)', 
+                'Time Elapsed 10 (milliseconds)', 
+                'Average of average CPU usage (% of CPU)', 
+                'Variance of average CPU usage (% of CPU)', 
+                'CPU usage 1 (% of CPU)', 
+                'CPU usage 2 (% of CPU)', 
+                'CPU usage 3 (% of CPU)', 
+                'CPU usage 4 (% of CPU)', 
+                'CPU usage 5 (% of CPU)', 
+                'CPU usage 6 (% of CPU)', 
+                'CPU usage 7 (% of CPU)', 
+                'CPU usage 8 (% of CPU)', 
+                'CPU usage 9 (% of CPU)', 
+                'CPU usage 10 (% of CPU)', 
+                'Average memory usage (% of memory)', 
+                'Variance of memory usage (% of memory)', 
+                'memory usage 1 (% of memory)', 
+                'memory usage 2 (% of memory)', 
+                'memory usage 3 (% of memory)', 
+                'memory usage 4 (% of memory)', 
+                'memory usage 5 (% of memory)', 
+                'memory usage 6 (% of memory)', 
+                'memory usage 7 (% of memory)', 
+                'memory usage 8 (% of memory)', 
+                'memory usage 9 (% of memory)', 
+                'memory usage 10 (% of memory)']]
 
 global benchCounter
 
 def measureMemoryUsage():
-	return memory_logging()
+    return memory_logging()
 
 def loadImageBench(imgPath):
-	return cv2.imread(image_path, cv2.IMREAD_GRAYSCALE)
+    return cv2.imread(image_path, cv2.IMREAD_GRAYSCALE)
 
+def contrastRandomization(img):
+    #Ranges for alpha and beta from this page: http://man.hubwiz.com/docset/OpenCV.docset/Contents/Resources/Documents/d3/dc1/tutorial_basic_linear_transform.html
+    alpha = random.uniform(1.0, 3.0)
+    beta = random.uniform(0, 100)
+    cv2.convertScaleAbs(img, alpha=2.2, beta=beta)
+	
+def brightnessRandomization(img):
+    value = random.uniform(0, 100)
+    hsv = cv2.cvtColor(img, cv2.COLOR_BGR2HSV)
+    hsv = np.array(hsv, dtype = np.float64)
+    hsv[:,:,1] = hsv[:,:,1]*value
+    hsv[:,:,1][hsv[:,:,1]>255]  = 255
+    hsv[:,:,2] = hsv[:,:,2]*value 
+    hsv[:,:,2][hsv[:,:,2]>255]  = 255
+    hsv = np.array(hsv, dtype = np.uint8)
+    cv2.cvtColor(hsv, cv2.COLOR_HSV2BGR)
```

Comment:
Looks good! (For future reviewers, I believe this is the only major change in this PR. The rest of the PR changes from tabs to spaces and does not need to be looked at.)